.env.staging

```diff
@@ -6,3 +6,6 @@ CP_ANALYTICS_API=https://cp-analytics-staging.reevoocloud.com/api/v1/
 # Tableau
 TABLEAU_GATEWAY_API=https://tableauapigateway-staging.reevoocloud.com/api/v1/
 TABLEAU_HOST=https://tableau-testing.reevoo.com/
+GTM_ENV=env-21
+GTM_AUTH_KEY=HkGbzImqRSXBnzwbiTCqaw
```

Comment:
@osa911 Is this fake or test Auth Key??

Reply:
It's real auth key for stage1 env in GTM